examples/r-job/.saturn/saturn.json

```diff
@@ -2,7 +2,7 @@
   "name": "example-r-job",
   "image_uri": "saturncloud/saturn-rstudio:2021.11.10",
   "description": "Run a job on Saturn Cloud using R .",
-  "working_directory": "/home/jovyan/git-repos/examples/examples/r-job",
+  "working_directory": "/home/jovyan/examples/examples/r-job",
```

Comment:
Can you remove this and make it a separate PR?

Reply:
I haven't removed this since otherwise it will throw validation error.